test/test.py

```diff
@@ -206,76 +125,66 @@ def test_get_pressure(mechname, usr_np):
 @pytest.mark.parametrize("mechname", [
     "uiuc", "sandiego", "uconn32", "gri30", "hong"
 ])
-@pytest.mark.parametrize("usr_np", numpy_list)
-def test_get_thermo_properties(mechname, usr_np):
+@pytest.mark.parametrize("user_np", numpy_list)
+def test_get_thermo_properties(mechname, user_np, request):
```

Comment:
@henryleberre can you add a description of `request` to the docstring please?

Reply:
`request` is an object that `pytest` passes us. I added the type-hint.

Reply:
But the docstrings is there, right?

Reply:
Why would there be a docstring? It feels to me like adding a docstring to a method explaining that the first argument, `self`, was of the same type as the class the method is in. This is the case because pytest passes it directly to us (we don't tell it to).

Reply:
I meant to ask if there was documentation on `request`, which there is. Thanks for adding it.